zorg/buildbot/builders/BOLTBuilder.py

```diff
@@ -130,6 +130,16 @@ def getBOLTCmakeBuildFactory(
                 haltOnFailure=False,
                 flunkOnFailure=False,
                 env=env),
+            ShellCommand(
+                name='llvm-bolt-version-check',
+                command=(f"{boltNew} --version"),
+                description=('Check that llvm-bolt binary passes a simple test'
+                             'before proceeding with testing.'),
+                descriptionDone=["llvm-bolt --version"],
+                haltOnFailure=True,
+                flunkOnFailure=True,
+                maxTime=30,
```

Comment:
This part has the whole worker to itself doesn't it?

(just confirming because time limits are ineffective when you have lots of processes fighting for resources)

Reply:
Correct, the worker processes one job at a time.